CONTRIBUTING.md

```diff
@@ -32,6 +32,18 @@ branch that doesn't have its origin in a pull request. We believe that not
 only does this improve the quality of our code base tremendously, but also 
 offers us a good way to learn from our mistakes.
 
+## Coding Conventions
+As coding conventions we adopted the Oracle's official [Conding conventions for
+the Java Programming Language]
+(http://www.oracle.com/technetwork/java/javase/documentation/codeconvtoc-136057.html)
+with the two follownig exceptions:
+
+* We use a line width of 120 characters.
+* We indent with four spaces and no tabs.
+
+you can find a formatter definition for the Eclipse IDE [here].
```

Comment:
The full stop at the end belongs after the link in the parenthesis, otherwise no link will be created.